web/crux/prisma/schema.prisma

```diff
@@ -308,6 +309,14 @@ enum RegistryTypeEnum {
   google
 }
 
+enum RegistryNamesapaceEnum {
+  unknown
```

Comment:
is unknown ever happens?

Reply:
yes in case of v2, dockerhub, and google